test.sh

```diff
@@ -1,20 +1,19 @@
 set -e
 
+# if greenplum supported (gpfdist exist), then test GP
+if hash gpfdist 2>/dev/null; then
+  echo "Testing Greenplum..."
+  bash test/test_gp.sh $1
 # if command psql exist, then test psql
-if hash psql 2>/dev/null; then
+elif hash psql 2>/dev/null; then
```

Comment:
@zifeishan : I see. I didn't get that. Ok, I'm going to revert it back to the original, but I'll add a comment explaining what you told me.